src/ontology/cl-edit.owl

```diff
@@ -3058,6 +3058,7 @@ Declaration(Class(obo:CL_4042013))
 Declaration(Class(obo:CL_4042014))
 Declaration(Class(obo:CL_4042015))
 Declaration(Class(obo:CL_4042016))
+Declaration(Class(obo:CL_4047006))
```

Comment:
Be careful, [immature pericyte](https://github.com/obophenotype/cell-ontology/pull/2504) has the same ID, this will create conflicts in the future. If you add two cells in two different branches at the same time, you have to manually (in protege) change the IR/ID

Reply:
Immature pericyte has been updated to  CL:4047007